projects/core-ui/schematics/utils.ts

```diff
@@ -1,7 +1,6 @@
 import { Tree, SchematicsException, SchematicContext, Rule } from '@angular-devkit/schematics';
 import { UkisNgAddSchema } from './ng-add/schema';
 import { NodePackageInstallTask } from '@angular-devkit/schematics/tasks';
-
 import { getWorkspace } from '@schematics/angular/utility/config';
```

Comment:
This will be removed in angular 11, will be temporarily fixed in the next PR  [branch update-angular](https://github.com/dlr-eoc/ukis-frontend-libraries/tree/update-angular)